lib/services/access_request.go

```diff
@@ -416,6 +416,13 @@ func ApplyAccessReview(req types.AccessRequest, rev types.AccessReview, author U
 
 	req.SetReviews(append(req.GetReviews(), rev))
 
+	if rev.AssumeStartTime != nil {
+		if rev.AssumeStartTime.After(req.GetAccessExpiry()) {
+			return trace.BadParameter("assumeStartTime is after request AccessExpiry")
```

Comment:
If this message can be seen by a user, let's use user-friendly language instead of developer-friendly language.
```suggestion
			return trace.BadParameter("request start time is after expiry")
```